integrated_channels/canvas/models.py

```diff
@@ -89,13 +89,15 @@ class CanvasEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfiguratio
 
     canvas_company_id = models.CharField(
         max_length=255,
+        null=True,
         blank=True,
         verbose_name="Canvas Organization Code",
         help_text="The organization code provided to the enterprise customer by Canvas."
     )
 
     canvas_base_url = models.CharField(
```

Comment:
This can be something we adjust later I guess, but do we plan on this being a per-customer value, because if not, we probably shouldn't have a database field for it? Like is it going to include their account number or something or is it just going to be basically `canvas.com` for everybody?

Reply:
I am pretty sure we will have a base_url per customer.. since it's their running instance of canvas (or a url they give us from a hosted provider of some kind)

Reply:
also if two customers happen to provide the same base url somehow, I guess we will still handle cleanly since we will use account_id or oauth tokens to do the rest ?

Reply:
Ah I assumed it was just uniformly hosted (aka saas), but you're absolutely right they could have it hosted anywhere.

Re uniqueness of the url:
As long as the base_url is not expected to be unique nor is it being used as a primary key it should be fine. I don't see any field options being used here that would indicate it as a unique field, so that should be fine if they do have the same instance url.

Reply:
Yeah, I think this is customer dependent but not necessarily unique.